Amazon A2I Fraud Detector.ipynb

```diff
@@ -0,0 +1,796 @@
+    "    '''\n",
+    "    Creates a Human Task UI resource.\n",
+    "\n",
+    "    Returns:\n",
+    "    struct: HumanTaskUiArn\n",
+    "    '''\n",
+    "    response = sagemaker.create_human_task_ui(\n",
+    "        HumanTaskUiName=task_ui_name,\n",
+    "        UiTemplate={'Content': template})\n",
+    "    return response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an Augmented AI task UI"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Task UI name - this value is unique per account and region. You can also provide your own value here.\n",
+    "taskUIName = 'fraud'+ str(uuid.uuid1())\n",
+    "\n",
+    "# Create task UI\n",
+    "humanTaskUiResponse = create_task_ui(taskUIName, template)\n",
+    "humanTaskUiArn = humanTaskUiResponse['HumanTaskUiArn']\n",
+    "print(humanTaskUiArn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "OUTPUT_PATH = f's3://{BUCKET}/a2i-results'\n",
+    "\n",
+    "def create_flow_definition(flow_definition_name):\n",
+    "    '''\n",
+    "    Creates a Flow Definition resource\n",
+    "\n",
+    "    Returns:\n",
+    "    struct: FlowDefinitionArn\n",
+    "    '''\n",
+    "    response = sagemaker.create_flow_definition(\n",
+    "            FlowDefinitionName= flow_definition_name,\n",
+    "            RoleArn= ROLE,\n",
+    "            HumanLoopConfig= {\n",
+    "                \"WorkteamArn\": WORKTEAM_ARN,\n",
+    "                \"HumanTaskUiArn\": humanTaskUiArn,\n",
+    "                \"TaskCount\": 1,\n",
+    "                \"TaskDescription\": \"Please review the claim data and flag for potential fraud\",\n",
+    "                \"TaskTitle\": \"Review and Approve / Reject claim.\"\n",
+    "            },\n",
+    "            OutputConfig={\n",
+    "                \"S3OutputPath\" : OUTPUT_PATH\n",
+    "            }\n",
+    "        )\n",
+    "    \n",
+    "    return response['FlowDefinitionArn']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Flow definition name - this value is unique per account and region. You can also provide your own value here.\n",
+    "#uniqueId = str(uuid.uuid4())\n",
+    "uniqueId = str(int(round(time.time() * 1000)))\n",
+    "flowDefinitionName = f'fraud-detector-a2i-{uniqueId}'\n",
+    "#flowDefinitionName = 'fraud-detector-a2i' \n",
+    "\n",
+    "flowDefinitionArn = create_flow_definition(flowDefinitionName)\n",
+    "print(flowDefinitionArn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Detect Moderation Labels with AWS Rekognition\n",
+    "\n",
+    "Let's call Amazon Rekognition to detect moderation labels on the sample images stored in S3 based on the steps above."
```

Comment:
Please remove this and all references to Rekognition

Reply:
incorporated in new pull request